src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -972,6 +972,49 @@ static void M()
             TestOperatorKinds(source);
         }
 
+        [Fact, WorkItem(27800, "https://github.com/dotnet/roslyn/issues/27800")]
+        public void TestDynamicCompoundOperatorOrdering()
+        {
+            CompileAndVerify(@"
+using System;
+class DynamicTest
+{
+	public int Property
```

Comment:
>	 [](start = 0, length = 1)

Please use spaces, not tabs.

Reply:
That's what I get for just copying the example off github.

---
In reply to: [199892616](https://github.com/dotnet/roslyn/pull/28247#discussion_r199892616) [](ancestors = 199892616)

Reply:
I still see tabs.

---
In reply to: [200424193](https://github.com/dotnet/roslyn/pull/28247#discussion_r200424193) [](ancestors = 200424193,199892616)